city_scrapers/spiders/chi_ssa_50.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa50Spider(CityScrapersSpider):
+    name = "chi_ssa_50"
+    agency = "Chicago Special Service Area"
+    timezone = "America/Chicago"
+    start_urls = ["http://southeastchgochamber.org/special-service-area-50/"]
+    base_xpath = '//div[@class="entry-content"]'
+    item_xpath = base_xpath + '//p[@style="text-align: center;"]/text()'
+
+    def parse(self, response):
+        title = self._parse_title(response)
+        description = self._parse_description(response)
+
+        for item in response.xpath(self.item_xpath).getall():
+            meeting = Meeting(
+                title=title,
+                description=description,
+                classification=self._parse_classification(item),
```

Comment:
These will all be the same, and it should be the constant `COMMISSION`

Reply:
The description should be the constant or an empty string?

Reply:
Sorry, this was just about the classification which should be `COMMISSION`. The description should be an empty string

Reply:
The description should be the constant or an empty string?